packages/labs/ssr-react/src/lib/node/render-custom-element.ts

```diff
@@ -37,6 +37,10 @@ export const renderCustomElement = (tagName: string, props: {} | null) => {
     deferHydration: false,
   };
 
+  // elementAttributes will be provided to React as props for the host element
+  // for properly rendering reflected attributes
+  const elementAttributes: {[index: string]: string} = {};
```

Comment:
Ultra nit - can ignore. What is `index`? Would the name `attributeName` make sense?